### DISCUSSION TITLE
Method to set stream from remote

### DISCUSSION THREAD

**Discussion by markretallack** [MatrixEditor/fsapi-tools#13] [General]
Hi, 

Does anyone know of an API to set the current stream to play?

I can use netRemote.nav.action.selectPreset to set a preset Internet radio stream to play, and I can use netRemote.nav.presets with the LIST_SET method to set a key, name and url, however is there a way to tell the radio to use a URL to play without having to set it as a preset first?

**Comment by MatrixEditor** [comment]
Unfortunately, there is no other way of playing a raw URL using the FSAPI. Since these devices store an UPnP server, it could be possible to play media streams using an URL directly (if this action is supported).
